rbx_xml/src/types/qdir.rs

```diff
@@ -0,0 +1,42 @@
+use std::io::{Read, Write};
+
+use crate::{
+    core::XmlType,
+    deserializer_core::XmlEventReader,
+    error::{DecodeError, EncodeError},
+    serializer_core::XmlEventWriter,
+};
+
+use rbx_dom_weak::types::QDir;
```

Comment:
```suggestion
use rbx_dom_weak::types::QDir;

use crate::{
    core::XmlType,
    deserializer_core::XmlEventReader,
    error::{DecodeError, EncodeError},
    serializer_core::XmlEventWriter,
};
```

We usually order external crates above internal imports.